adyen.php

```diff
@@ -223,50 +259,50 @@ public function getContent()
     {
         $output = null;
 
-        if (\Tools::isSubmit('submit' . $this->name)) {
+        if (Tools::isSubmit('submit' . $this->name)) {
             // get post values
-            $merchant_account = (string)\Tools::getValue('ADYEN_MERCHANT_ACCOUNT');
-            $mode = (string)\Tools::getValue('ADYEN_MODE');
-            $notification_username = (string)\Tools::getValue('ADYEN_NOTI_USERNAME');
-            $notification_password = (string)\Tools::getValue('ADYEN_NOTI_PASSWORD');
-            $notification_hmac = (string)\Tools::getValue('ADYEN_NOTI_HMAC');
-            $cron_job_token = \Tools::getValue('ADYEN_CRONJOB_TOKEN');
-            $api_key_test = \Tools::getValue('ADYEN_APIKEY_TEST');
-            $api_key_live = \Tools::getValue('ADYEN_APIKEY_LIVE');
-            $live_endpoint_url_prefix = (string)\Tools::getValue('ADYEN_LIVE_ENDPOINT_URL_PREFIX');
+            $merchant_account = (string)Tools::getValue('ADYEN_MERCHANT_ACCOUNT');
+            $mode = (string)Tools::getValue('ADYEN_MODE');
+            $notification_username = (string)Tools::getValue('ADYEN_NOTI_USERNAME');
+            $notification_password = (string)Tools::getValue('ADYEN_NOTI_PASSWORD');
+            $notification_hmac = (string)Tools::getValue('ADYEN_NOTI_HMAC');
+            $cron_job_token = Tools::getValue('ADYEN_CRONJOB_TOKEN');
+            $api_key_test = Tools::getValue('ADYEN_APIKEY_TEST');
+            $api_key_live = Tools::getValue('ADYEN_APIKEY_LIVE');
+            $live_endpoint_url_prefix = (string)Tools::getValue('ADYEN_LIVE_ENDPOINT_URL_PREFIX');
 
             // validating the input
-            if (!$merchant_account || empty($merchant_account) || !\Validate::isGenericName($merchant_account)) {
+            if (!$merchant_account || empty($merchant_account) || !Validate::isGenericName($merchant_account)) {
```

Comment:
```suggestion
            if (empty($merchant_account) || !Validate::isGenericName($merchant_account)) {
```

If we remove the `!$merchant_account` the result will be always the same or we need to check it specifically for something?